django/thunderstore/security/utils.py

```diff
@@ -0,0 +1,140 @@
+import base64
+import binascii
+import json
+
+import requests
+from cryptography.hazmat.backends.openssl.backend import backend
+from cryptography.hazmat.primitives.asymmetric import ec
+from six import b, text_type
```

Comment:
Not necessary, we're Python 3 only so we don't need to use python 2 compatibility features